stytch/b2b/organizations/types.go

```diff
@@ -0,0 +1,463 @@
+//     provisioned upon authentication via Email Magic Link.
+//
+//     `NOT_ALLOWED` – disable JIT provisioning via Email Magic Link.
+//
+//   - EmailInvites: The authentication setting that controls how a new Member can be invited to an
+//     organization by email. The accepted values are:
+//
+//     `ALL_ALLOWED` – any new Member can be invited to join via email.
+//
+//     `RESTRICTED` – only new Members with verified emails that comply with `email_allowed_domains` can be
+//     invited via email.
+//
+//     `NOT_ALLOWED` – disable email invites.
+//
+//   - AuthMethods: The setting that controls which authentication methods can be used by Members of an
+//     Organization. The accepted values are:
+//
+//     `ALL_ALLOWED` – the default setting which allows all authentication methods to be used.
+//
+//     `RESTRICTED` – only methods that comply with `allowed_auth_methods` can be used for authentication.
+//     This setting does not apply to Members with `is_breakglass` set to `true`.
+//
+//   - AllowedAuthMethods:
+//     An array of allowed authentication methods. This list is enforced when `auth_methods` is set to
+//     `RESTRICTED`.
+//     The list's accepted values are: `sso`, `magic_link`, and `password`.
+//
+//   - TrustedMetadata: An arbitrary JSON object for storing application-specific data or
+//     identity-provider-specific data.
+//
+//   - SsoDefaultConnectionID: The default connection used for SSO when there are multiple active
+//     connections.
+type Organization struct {
+	OrganizationID                       string                `json:"organization_id,omitempty"`
+	OrganizationName                     string                `json:"organization_name,omitempty"`
+	OrganizationLogoURL                  string                `json:"organization_logo_url,omitempty"`
+	OrganizationSlug                     string                `json:"organization_slug,omitempty"`
+	SsoJitProvisioning                   string                `json:"sso_jit_provisioning,omitempty"`
+	SsoJitProvisioningAllowedConnections []string              `json:"sso_jit_provisioning_allowed_connections,omitempty"`
+	SsoActiveConnections                 []ActiveSSOConnection `json:"sso_active_connections,omitempty"`
+	EmailAllowedDomains                  []string              `json:"email_allowed_domains,omitempty"`
+	EmailJitProvisioning                 string                `json:"email_jit_provisioning,omitempty"`
+	EmailInvites                         string                `json:"email_invites,omitempty"`
+	AuthMethods                          string                `json:"auth_methods,omitempty"`
+	AllowedAuthMethods                   []string              `json:"allowed_auth_methods,omitempty"`
+	TrustedMetadata                      map[string]any        `json:"trusted_metadata,omitempty"`
+	SsoDefaultConnectionID               string                `json:"sso_default_connection_id,omitempty"`
```

Comment:
This value is a pointer - it can be `null` when no default conn is set, instead of the empty string